DigitalLearningSolutions.Data/Models/CourseContent/CourseContent.cs

```diff
@@ -1,12 +1,16 @@
 ﻿namespace DigitalLearningSolutions.Data.Models.CourseContent
 {
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Data.Models.Courses;
+
     public class CourseContent
     {
         public int Id { get; private set; }
         public string Title { get; private set; }
         public string AverageDuration { get; private set; }
         public string CentreName { get; private set; }
         public string? BannerText { get; private set; }
+        public List<CourseSection> Sections { get; private set; } = new List<CourseSection>();

```

Comment:
Something I noticed when looking through the code yesterday, you can just remove the `private set` as that's equivalent to not having a set at all. The IDE should warn you of this by showing it as greyed out 🙂

Reply:
Yeah I think misunderstood one of your reviews for #37 / HEEDLS-203 here. Fixed in 7c72149 🙂